doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows the updating of both the application core and the network core in one go.
+
+To enable the simultaneous update of multiple images, set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion, so you need to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.
+
+Additionally, the memory partitions for secondary images for both the application and the network core must be defined.
```

Comment:
I would say in general that memory partitions must be defined. We also define RAM partitions for PCD SRAM and primary partition for netcore in MCUboot (the one that uses simulated flash)

Reply:
Please see how it looks now.